python/lsst/ts/mtdome/mock_llc/mock_motion/base_llc_motion_with_crawl.py

```diff
@@ -19,14 +19,15 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import abc
 import math
 
 from lsst.ts.idl.enums.MTDome import MotionState
 
 from .base_llc_motion import BaseLlcMotion
 
 
-class BaseLlcMotionWithCrawl(BaseLlcMotion):
+class BaseLlcMotionWithCrawl(BaseLlcMotion, abc.ABC):
```

Comment:
Why this change? I thought abstraction was inherited.

Reply:
I added because PyCharm complains if it isn't there. But I removed it again because it, indeed, is inherited.

Reply:
Thank you for the explanation. Sorry about PyCharm.